Assets/LeapC/LeapC.cs

```diff
@@ -388,24 +388,27 @@ public LEAP_VECTOR(UnityEngine.Vector3 unity)
   }
 
   [StructLayout(LayoutKind.Sequential, Pack = 1)]
-  public struct LEAP_MATRIX
+  public struct LEAP_QUATERNION
   {
-    public LEAP_VECTOR x_basis;
-    public LEAP_VECTOR y_basis;
-    public LEAP_VECTOR z_basis;
+    public float w;
+    public float x;
+    public float y;
+    public float z;
 
-    public Leap.Matrix ToLeapMatrix()
+    public LEAP_QUATERNION(Leap.LeapQuaternion q)
     {
-      return new Leap.Matrix(x_basis.ToLeapVector(),
-                             y_basis.ToLeapVector(),
-                             z_basis.ToLeapVector());
+      w = q.w;
+      x = q.x;
+      y = q.y;
+      z = q.z;
     }
 
-    public LEAP_MATRIX(Leap.Matrix leap)
+    public LEAP_QUATERNION(UnityEngine.Quaternion unity)
```

Comment:
Do we need an if defined UnityEngine block here so that it compiles outside of Unity?